### DISCUSSION TITLE
How to install Arcana wallet ?

### DISCUSSION THREAD

**Discussion by gzliudan** [arcana-network/discussions#6] [General]
Hello, how to install Arcana wallet ?

**Comment by shaloo** [comment]
Hi,

Arcana wallet is an embedded non-custodial web wallet that is part of the Arcana Auth SDK. It is not a standalone wallet. To enable Arcana wallet in an app, you need to install and integrate your app with the Arcana Auth SDK.

Refer to the documentation for registering, configuring and integrating your app with the Auth SDK.

What is your use case?

On 06-Apr-2023, at 4:04 PM, Daniel Liu ***@***.***> wrote:



Hello, how to install Arcana wallet ?

**Comment by gzliudan** [comment]
How to support an new blockchain ?  such as:
- mainnet: https://chainlist.org/chain/50
- testnet: https://chainlist.org/chain/51

**Reply by shaloo** [reply]
Hi there! 

Currently, we support EVM-compatible chains. See how to add/switch networks [here](https://docs.arcana.network/howto/arcana_wallet/wallet_add_switch_ntwk.html#prerequisites): and via wallet UI - [here](https://docs.arcana.network/howto/wallet_ui.html#addselect-a-network).

**Reply by gzliudan** [reply]
The new EVM-compatible  blockchain should be added by APP developer or by APP end users ?

**Reply by gzliudan** [reply]
Should the blockchain support [EIP-3085](https://eips.ethereum.org/EIPS/eip-3085) and [EIP-3326](https://eips.ethereum.org/EIPS/eip-3326) ?

**Reply by shaloo** [reply]
> Both.
Devs can add them programmatically (see doc links to the Wallet developer's guide that I shared earlier).  Users can add them via the Wallet UI.  However, the user's chain updates do not persist across app/wallet sessions.  See the warning in the footer section '[here](https://docs.arcana.network/concepts/anwallet/walletntwkswitchmode.html#network-switching-alwaysvisibletrue)'.

**Comment by mmjee** [comment]
On 4/7/23 08:27, Daniel Liu wrote:
> If blockchain does not support [EIP-3085](https://eips.ethereum.org/EIPS/eip-3085) and [EIP-3326](https://eips.ethereum.org/EIPS/eip-3326), does it can be added in APP ?
> 
Yes, EIP-3085 and EIP-3326 is by nature necessary for switching chains. Furthermore, it's likely very easy to support it anyway. If the chain is EVM-supported and has a chain ID configured (see whether the “eth_chainId” JSON-RPC method works), it should work as usual.

The app. developer needs to request to add it (through EIP-3085) and then request changing to it (through EIP-3326).
-- 
Regards,

**Reply by gzliudan** [reply]
It seems that [EIP-3085](https://eips.ethereum.org/EIPS/eip-3085) and [EIP-3326](https://eips.ethereum.org/EIPS/eip-3326) are implemented by wallet, not by blockchain ?

**Comment by mmjee** [comment]
On 4/7/23 14:39, Daniel Liu wrote:
> It seems that [EIP-3085](https://eips.ethereum.org/EIPS/eip-3085) and [EIP-3326](https://eips.ethereum.org/EIPS/eip-3326) are implemented by wallet, not by blockchain ?
> 
It is, but the underlying chain needs to support a chain ID. We do support EIP-3085 and EIP-3326. If there's a chain ID configured for the chain, it should work as usual.
-- 
Regards,

**Comment by gzliudan** [comment]
Ok, I understand now. Thank you.
